WzComparerR2.Common/Text/Parser.cs

```diff
@@ -65,6 +65,12 @@ public static IList<DocElement> Parse(string format)
                             colorStack.Push("c");
                             strPos++;
                         }
+                        else if (strPos < format.Length && format[strPos] == 'e')//遇到#e(自定义) 换紫蓝刷子并flush
```

Comment:
As we disucssed offline, we SHOULD NOT add more custom Macros, because it may not be compatible with abbreviated #....# section for GMS/MSEA clients.

In future we can extend #$ as custom macros, and also remove `#g` if possible.

Reply:
Commits other than ScrollEnhancementColor definition were reverted just now.